src/styles/components/_override.buttons.scss

```diff
@@ -212,3 +221,10 @@
   font-family: $montserrat-font;
   font-weight: 600;
 }
+
+// Hides elements like buttons when printing
+@media print {
+  .mds-hide-print {
```

Comment:
@bkerstan @nichia Thank you for all the feedback. Do you think there is a better class name for this class as well? Maybe .mds-u-hide-print?

Reply:
That's a good point. We generally stay with this [code convention](https://design.cms.gov/guidelines/code-conventions/).
But as an aside, I noticed there exists a [media print .ds-u-visibility--hidden-print](https://github.com/CMSgov/design-system/blob/master/packages/design-system/src/styles/utilities/_display-visibility.scss#L44). Can this be use instead of mds-hide-print?